### DISCUSSION TITLE
cp: missing destination file operand after '/home/ofuser/blueCFD/OpenFOAM-5.x/tutorials/incompressible/icoFoam/cavity/cavity'

### DISCUSSION THREAD

**Discussion by mustafees** [blueCFD/Core#230] [Q&A] [answered]
blue CFD is not able to find the directories in run. I copied all the files from another in the user of 5 folder to no avail. Followed the same step by step installation procedure.  The terminal leads to errors

**Comment by wyldckat** [comment, marked as answer]
Without an image to see what you are seeing, it is very difficult to deduce what went wrong.

From the subject line of the report you have opened:
```
cp: missing destination file operand after '/home/ofuser/blueCFD/OpenFOAM-5.x/tutorials/incompressible/icoFoam/cavity/cavity'
```

This means that you forget to tell where to copy it to, namely you must state where to copy to. For example, these commands should:
```
mkdir -p $FOAM_RUN
cd $FOAM_RUN
cp -r $FOAM_TUTORIALS/incompressible/icoFoam/cavity/cavity ./
```

Notice at the end of the last line is this: `./`
